README.md

```diff
@@ -134,12 +134,13 @@ gulp.task('pre-push', guppy.src('pre-push', function (files, extra, cb) {
 }));
 ```
 
-#### guppy.stream(*hookName*)
+#### guppy.stream(*hookName, options*)
```

Comment:
This needs to be `guppy.stream(*hookName[, options]*)` to show that options are *optional*.